charts/dkube-deployer/values.schema.json

```diff
@@ -60,13 +58,11 @@
 				},
 				"username":{
 					"$id":"#/properties/registry/properties/username",
-					"type":"string",
-					"minLength":1
```

Comment:
minLength is removed. any reason?

Reply:
@deepio-oc helm fails the install if minLength is set for a field and the value for the field in values.yaml is empty.

minLength had to be removed since we have use cases where username and password are empty (https://github.com/oneconvergence/dkube-helm/pull/18#issuecomment-1061868091)